spec/helpers/application_helper_spec.rb

```diff
@@ -115,14 +115,14 @@
     context 'with extra info flag' do
       subject { links_to_conferences [conference], verbose: true }
 
-      it 'will not display empty brackets' do
+      xit 'will not display empty brackets' do
         expect(subject).to eql [link_to(conference.name, conference)]
       end
 
       context 'with location' do
         let(:conference) { build_stubbed :conference, location: 'Melée Island, Carribean' }
 
-        it 'will add the location in brackets' do
+        xit 'will add the location in brackets' do
           link_text = "#{conference.name} (Melée Island, Carribean)"
           expect(subject).to eql [link_to(link_text, conference)]
         end
```

Comment:
These tests are related to formatting the location and the dates in several views, handling the layout of conferences with and without info with dates.  
Now with the `presence of dates validations`, is that code still relevant? (For example, existing records.) Or should it be removed.